packages/safe-ds-lang/src/language/runner/safe-ds-runner.ts

```diff
@@ -177,32 +202,39 @@ export class SafeDsRunner {
             this.messageCallbacks.get(messageType)!.filter((storedCallback) => storedCallback !== callback),
         );
     }
+    /* c8 ignore stop */
 
     /**
      * Get information about a pipeline execution.
      *
      * @param pipelineId Unique id that identifies a pipeline execution
      * @return Execution context assigned to the provided id.
      */
+    /* c8 ignore start */
     public getExecutionContext(pipelineId: string): PipelineExecutionInformation | undefined {
         return this.executionInformation.get(pipelineId);
     }
+    /* c8 ignore stop */
 
     /**
      * Remove information from a pipeline execution, when it is no longer needed.
      *
      * @param pipelineId Unique id that identifies a pipeline execution
      */
+    /* c8 ignore start */
     public dropPipelineExecutionContext(pipelineId: string) {
         this.executionInformation.delete(pipelineId);
     }
+    /* c8 ignore stop */
 
     /**
      * Remove information from all previous pipeline executions.
      */
+    /* c8 ignore start */
     public dropAllPipelineExecutionContexts() {
         this.executionInformation.clear();
     }
+    /* c8 ignore stop */
```

Comment:
You can also remove the intermediate stop/start commands and keep only the beginning and end ones. I wouldn't mind either, if you just put a `/* c8 ignore start */` at the start of the file with an explanatory comment.